docs-v2/content/en/docs/pipeline-stages/deployers/helm.md

```diff
@@ -96,6 +110,129 @@ deploy:
         helm: {}
 ```
 
+### Image reference strategies
+
+Skaffold supports three _image reference strategies_ for Helm:
+
+1. `fqn`: provides a fully-qualified image reference (default);
+2. `helm`: provides separate repository and tag portions (shown above);
+3. `helm+explicitRegistry`: provides separate registry, repository, and tag portions.
+
+#### `fqn` strategy: single fully-qualified name (default)
+
+With the fully-qualified name strategy, Skaffold configures Helm by setting a key to the fully-tagged image reference.
+
+The `skaffold.yaml` setup:
+```yaml
+build:
+  artifacts:
+    - image: gcr.io/my-project/my-image  # must match in setValues
+manifests:
+  helm:
+    releases:
+      - name: my-chart
+        chartPath: helm
+        setValues:
+          image: gcr.io/my-project/my-image
+```
+The `values.yaml` (note that Skaffold overrides this value):
+```
+image: gcr.io/other-project/other-image:latest
+```
+
+The chart template:
+```yaml
+spec:
+  containers:
+    - name: {{ .Chart.Name }}
+      image: "{{.Values.image}}"
+```
+
+Skaffold will invoke
+```
+helm install <chart> <chart-path> --set-string image=<artifact-name> --post-renderer=<path-to-skaffold-binary-from-original-invocation>
+```
+
+#### `helm` strategy: split repository and tag
+
+Skaffold can be configured to provide Helm with a separate repository and tag.  The key used in the `artifactOverrides` is used as base portion producing two keys `{key}.repository` and `{key}.tag`.
+
+The `skaffold.yaml` setup:
+```yaml
+build:
+  artifacts:
+    - image: gcr.io/my-project/my-image
+manifests:
+  helm:
+    releases:
+      - name: my-chart
+        chartPath: helm
+        artifactOverrides:
```

Comment:
Alert above says "No more `artifactOverrides` or `imageStrategy`", yet `artifactOverrides` is here? Same w/ `imageStrategy` down below. Were these options removed or are they now optional?

Reply:
Thank you for catching this, this was a bad copy paste and there should be no `artifactOverrides`.  I have updated this to be the correct set of `setValues` entries instead as it should have been